frontend/src/views/Home.vue

```diff
@@ -76,7 +77,7 @@ export default {
     }
   },
   mounted () {
-    if (this.$route.path === '/namespace/+') {
+    if (get(this.$route.query, 'projectDialog') === 'true') {
```

Comment:
add route with `+` and redirect as it is used by the `vscode-gardener-tools`

Reply:
✔️